semanticz-connector-neptune/build.gradle.kts

```diff
@@ -0,0 +1,16 @@
+plugins {
+    id("semanticz.library")
+}
+
+project.description = "Neptune connector"
+
+dependencies {
+    implementation(project(":semanticz-connectors-common"))
+    implementation(libs.vavr)
+    implementation(libs.httpclient)
+    implementation(libs.jena.arq)
+    implementation(libs.jena.rdfconnection)
+    testImplementation(libs.junit.jupiter)
+    testImplementation ("org.mockito:mockito-core:5.2.0")
```

Comment:
should we not follow the same convention? cfr "testImplementation(libs.junit.jupiter)"

I guess mockito is reaonably stable wrt its api. so we can upgrade across all modules (even if it is just one atm?)